playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -88,7 +18,7 @@
   - set_fact:
       etcd_backup_completed: "{{ hostvars
                                  | oo_select_keys(groups.oo_etcd_hosts_to_backup)
-                                 | oo_collect('inventory_hostname', {'etcd_backup_complete': true}) }}"
+                                 | oo_collect('inventory_hostname', {'r_etcd_upgrade_backup_complete': true}) }}"
```

Comment:
Maybe `o_etcd_upgrade_back_complete` to state the variable is outputed by `etcd_upgrade` role?